README.md

```diff
@@ -6,7 +6,55 @@ Accessible and customizable components that you can copy and paste into your app
 
 ## Documentation
 
-Visit http://ui.shadcn.com/docs to view the documentation.
+> **Warning**
+> This is work in progress. I'm building this in public. You can follow the progress on Twitter [@shadcn](https://twitter.com/shadcn).
+
+- [ ] Toast
+- [ ] Toggle Group
+- [ ] Toolbar
+- [x] ~Toggle~
+- [x] ~Navigation Menu~
+- [x] ~Figma~
+
+## Storybook
+
+Storybook is supported so that you can easily select the correct component and view its properties
+
+![Storybook Preview](apps/www/public/demo/storybook.png)
+
+To open Storybook, run the following:
+
+```bash
+cd apps/www
+npm i
+npm run storybook
+```
+
```

Comment:
Since the Shadcn/ui v0.2.3 the Readme is a lot more minimalistic.  Might be better to move these instruction from the Readme.md and instead into the actual app/www 🤷